### PR TITLE
Add presence for V2ex

### PR DESCRIPTION
**Pull request by MeXunco**
[V2EX](https://www.v2ex.com) is a community of start-ups, designers, developers and creative people for china.

<details>
  <summary>Click here to view screenshots</summary>
  
  
  
  
  
  
</details>

### CODE REVIEW

File: V2ex/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "en": "A Community of Creative Workers. Discuss exciting topics such as programming, design, hardware, games, and more.",
+    "nl": "Een gemeenschap van creatieve werkers. Bespreek spannende onderwerpen zoals programmeren, ontwerp, hardware en games.",
+    "zh_cn": "创意工作者的社区。讨论编程、设计、硬件、游戏等令人激动的话题。",
+    "zh_tw": "創意工作者的社區。討論編程、設計、硬件、遊戲等令人激動的話題。"
+  },
+  "url": ["www.v2ex.com", "fast.v2ex.com", "v2ex.com"],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/kfj4PUe.png",
+  "thumbnail": "https://i.imgur.com/eugm5S4.png",
+  "color": "#000000",
```
Comment (Alanexei) [inline V2ex/dist/metadata.json:17]: This isn't necessary, but can you find a color that accustomed to the website's theme? Such as a dark blue or dark gray.

Comment (MeXunco) [reply V2ex/dist/metadata.json:17]: Thanks, please wait.

**Comment by Bas950** [comment]
> image
> Delete this file

Your presence right now is:
presence
├── dist
│   ├── metadata.json
│   └── presence.js
├── presence.ts
├── presence.js
└── tsconfig.json

It should be:
presence
├── dist
│   ├── metadata.json
│   └── presence.js
├── presence.ts
└── tsconfig.json

**Comment by MeXunco** [comment]
I have changed that, thank you for pointing that out!



> > image
> > Delete this file
> 
> Your presence right now is:
> presence
> ├── dist
> │ ├── metadata.json
> │ └── presence.js
> ├── presence.ts
> ├── presence.js
> └── tsconfig.json
> 
> It should be:
> presence
> ├── dist
> │ ├── metadata.json
> │ └── presence.js
> ├── presence.ts
> └── tsconfig.json


I have changed that, thank you for pointing that out!

**Comment by Bas950** [comment]
:thonk:

**Comment by MeXunco** [comment]
Sorry, I made a mistake👼

**Comment by MeXunco** [comment]
It seems that you need to set up

> ERROR environment variable GITHUB_TOKEN not set! [click link](https://github.com/PreMiD/Presences/pull/1327/checks?check_run_id=590889675)

**Comment by doomlerd** [comment]
@Fruxh

**Pull request merged**